rusqlite_migration/src/lib.rs

```diff
@@ -600,12 +600,19 @@ impl<'m> Migrations<'m> {
         res
     }
 
-    /// Maximum version defined in the migration set
-    fn max_schema_version(&self) -> SchemaVersion {
+    /// Return the maximum version defined in the migration set.
+    ///
+    /// If you are using [`Migrations::from_directory()`] and `include_dir!`, it is non-trivial to
+    /// access the number of migrations loaded, so you may need to use this method.
+    ///
+    /// For the most common scenarios though, you should be able to just call
+    /// [`Migrations::to_latest`].
```

Comment:
@fluffysquirrels anything you would change in this documentation?

Reply:
I tweaked the wording slightly, but this looks good to me!